core/management/commands/queue_warm_cache.py

```diff
@@ -0,0 +1,22 @@
+import logging
+
+from django.core.management.base import BaseCommand
+from django.core.management.base import CommandError
+
+from chronam.core.models import Page
+from chronam.core import tasks
+
+_logger = logging.getLogger(__name__)
+
+class Command(BaseCommand):
+    help = "Queue a celery request for each page from chronicling america to warm up the cache."
+
+    def handle(self, **options):
+        for page in Page.objects.all():
+	    try:
+                url = "https://chroniclingamerica.loc.gov%s" % page.url
+                _logger.info("queuing url %s"% url)
+                tasks.load_page_in_cache.delay(url)
+            except Exception, e:
```

Comment:
Use the Python 2.6+ `except … as var_name` syntax — it'll be necessary for Python 3 and while it's not relevant in this specific case as a habit it avoids the confusion which was possible in the past when catching more than one type of exception.

[modernize](https://pypi.python.org/pypi/modernize) can do this for you automatically:
```diff
cadams@Ganymede:~/Projects/chronam [git warm_cache] $ python-modernize --no-six core/management/commands/queue_warm_cache.py 
RefactoringTool: Skipping optional fixer: idioms
RefactoringTool: Skipping optional fixer: set_literal
RefactoringTool: Skipping optional fixer: ws_comma
RefactoringTool: Refactored core/management/commands/queue_warm_cache.py
--- core/management/commands/queue_warm_cache.py	(original)
+++ core/management/commands/queue_warm_cache.py	(refactored)
@@ -1,3 +1,4 @@
+from __future__ import absolute_import
 import logging
 
 from django.core.management.base import BaseCommand
@@ -17,6 +18,6 @@
                 url = "https://chroniclingamerica.loc.gov%s" % page.url
                 _logger.info("queuing url %s"% url)
                 tasks.load_page_in_cache.delay(url)
-            except Exception, e:
+            except Exception as e:
                 _logger.exception(e)
                 raise CommandError("Unable to queue warm cache request. Check the log for clues.")
RefactoringTool: Files that need to be modified:
RefactoringTool: core/management/commands/queue_warm_cache.py
```